content/spire/getting-started.md

```diff
@@ -1,203 +1,353 @@
+
+* What you will name your server trust domain and your agent trust domain 
+
+Trust domain names must be identical in the server and the agent. 
+
+* Which node attestation method your application requires 
+
+This depends on your where your workload is running. Your choice of node attestation method determines which node-attestor plugins you configure SPIRE to use in Server Plugins and Agent Plugins sections of the SPIRE configuration files. You must configure at least one node attestor on the server and only one node attestor on the agent. 
+
+For simplicity’s sake, this guide demonstrates using the join token method for node attestation. 
+
+* Which workload attestation method your application requires. As with node attestation methods, your choice depends on the infrastructure your application’s workloads are deployed in (for example, SPIRE supports identifying workloads that run in Kubernetes).
+
+* Which type of database your application will use to store server data
+
+SPIRE employs a database to persist data related to workload identities and registration entries. By default, SPIRE bundles SQLite and sets it as the default for storage of server data. SPIRE currently also supports PostgreSQL. For production purposes, you should carefully consider which database to use. 
+
+* Which key management backend your application requires
+
+The key manager generates and persists the public-private key pair used for the agent SVID.  You must choose whether to store the private key on disk or in memory. For production purposes, you also might consider integrating a custom backend for storage purposes, such as a secret store.
+
+* Which trust root (“upstream certificate authority (CA)”) your application will use
+
+The SPIRE server provides a CA. If you’re, for example, using an external PKI system
+	that provides an upstream CA. you can configure SPIRE to use that instead.
+
+Once you’ve made these decisions, you can [configure the server](#step-4-configure-the-server) and [configure the agent](#step-5-configure-the-agent) accordingly, after [installing them](#step-3-install-the-server-and-agent). 
+
+## Step 2 Obtain the SPIRE Binaries
+
+Pre-built SPIRE releases can be found in the [GitHub SPIRE releases directory](https://github.com/spiffe/spire/releases). These releases contain both server and agent binaries.
+
+If you wish, you may also [build SPIRE from source](https://github.com/spiffe/spire/blob/master/CONTRIBUTING.md).
+
+## Step 3 Install the Server and Agent
+
+As stated above, this guide illustrates installs the server and agent on the same node. More typically, your architecture will have the the server installed on one node and one or more agents installed on distinct nodes. 
+
+To install the server and agent:
+
+1. Obtain the latest tarball from [the GitHub SPIRE releases directory](https://github.com/spiffe/spire/releases): 
+
+and then extract it into the **/opt/spire** directory using the following commands:
+
+``
```

Comment:
Code blocks like this should be:

1. With triple backticks, not double
2. No indentation
3. Should specify a language, in this case `shell`

Here's an example:

```
```shell
sudo tar zvxf spire-0.7.0-linux-x86_64-glibc.tar.gz
sudo cp -r spire-0.7.0/. /opt/spire/
```

Also, don't use `#` as the CLI prompt, as that will comment out the text.

Reply:
Fixed.